policyasn1/policy.go

```diff
@@ -1,21 +1,16 @@
 // policyasn1 contains structures required to encode the RFC 5280
-// PolicyInformation ASN.1 structures.
+// Certificate Policies extension ASN.1 structures.
 package policyasn1
 
 import "encoding/asn1"
 
-// CPSQualifierOID contains the id-qt-cps OID that is used to indicate the
-// CPS policy qualifier type
-var CPSQualifierOID = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 2, 1}
+// CertificatePoliciesExtOID is the OID which identifies the Certificate
+// Policies extension, defined as id-ce-certificatePolicies in RFC 5280.
+var CertificatePoliciesExtOID = asn1.ObjectIdentifier{2, 5, 29, 32}
 
-// PolicyQualifier represents the PolicyQualifierInfo ASN.1 structure
-type PolicyQualifier struct {
-	OID   asn1.ObjectIdentifier
-	Value string `asn1:"optional,ia5"`
-}
-
-// PolicyInformation represents the PolicyInformation ASN.1 structure
+// PolicyInformation represents the PolicyInformation ASN.1 structure. It
+// excludes the Qualifiers field because that field is NOT RECOMMENDED for all
+// certificate profiles in the BRs.
 type PolicyInformation struct {
```

Comment:
Could this file move into `//policy/asn1/policy.go`? But then `//cmd/ceremony/cert.go` and `//issuance/issuance.go` would need to reference that location instead and what did it really gain us other than shuffling files around.

Reply:
And confusingly, the "policy" in the "//policy" directory and the "policy" in "policyasn1" actually have nothing to do with each other: the former is talking about the policies we place on what names we're willing to issue for, and the latter refers to the "Certificate Policies Extension". Okay I guess they're _kinda_ related but nothing else in the //policy directory has to do with certificate encoding, just issuance decisions.